src/app/components/PortraitVideoCarousel/fixture.ts

```diff
@@ -108,7 +108,7 @@ const portraitVideoFixture = {
       },
     },
     {
-      id: 'urn:bbc:pips:pid:p01wjx7v',
+      id: 'urn:bbc:pips:pid:p01wjx7v-copy',
```

Comment:
Did we mean to change this?

Reply:
Yeah, because otherwise we get this error:  "Each child in a list should have a unique "key" prop" 
The fixture data duplicates some of the promos to fill out the carousel.

Reply:
You could use a combination of index & id?

Reply:
I tried that too, but it still says: 'Do not use Array index in keys'

Reply:
That's annoying! Shall we just use a number instead of the word copy here?
```suggestion
      id: 'urn:bbc:pips:pid:p01wjx7v1',
```